src/main/java/Duke.java

```diff
@@ -1,10 +1,95 @@
+            String from = description.substring(j+6, k).trim();
+            String to = description.substring(k+4);
+            description = description.substring(0, j).trim();
+            tasks[taskCount] = new Events(description, from, to);
+            break;
+        }
+        taskCount++;
+        System.out.println("\tGot it. I've added this task:");
+        System.out.println("\t" + tasks[taskCount-1]);
+        System.out.println("\tNow you have " + taskCount + " task(s) in the list.");
+    }
+    public static void printTasks() {
+        if (taskCount == 0) {
+            System.out.println("\tYou do not have any task in the list.");
+            return;
+        }
+        System.out.println("\tHere are the tasks in your list:");
+        for (int i=0; i<taskCount; i++) {
+            System.out.print("\t");
+            System.out.print(i+1 + ".");
+            System.out.println(tasks[i]);
+        }
+    }
+    public static void markAsDone(int number) {
+        tasks[number-1].setDone(true);
+        System.out.println("\tNice! I've marked this task as done:");
+        System.out.println("\t" + tasks[number-1]);
+    }
+    public static void markAsUndone(int number) {
+        tasks[number-1].setDone(false);
+        System.out.println("\tNice! I've marked this task as not done yet:");
+        System.out.println("\t" + tasks[number-1]);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Greetings
+        String intro = "~~~~~~~~~~~~~~~~~~~\n"
+                        + "Hello! I'm Wenny!\n"
+                        + "How may I help you?\n"
+                        + "~~~~~~~~~~~~~~~~~~~";
+        System.out.println(intro);
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+            String userInput = scanner.nextLine();
+            String[] substr = userInput.split("\\s+");
+            switch (substr[0]) {
+            case "bye":
+                bye();
+                break;
+            case "list":
+                printTasks();
+                break;
+            case "mark":
+                markAsDone(Integer.parseInt(substr[1]));
+                break;
+            case "unmark":
+                markAsUndone(Integer.parseInt(substr[1]));
+                break;
+            case "todo":
+                addToTasks(userInput.substring(5), Type.TODO);
+                break;
+            case "deadline":
+                addToTasks(userInput.substring(9), Type.DEADLINE);
+                break;
+            case "event":
+                addToTasks(userInput.substring(6), Type.EVENT);
+                break;
+            }
+        }
+    }
+    public static void bye() {
```

Comment:
Perhaps you could use a more clear method name such as printByeMessage?
```suggestion
    public static void printByeMessage() {
```